services/blockchain-indexer/shared/dataService/business/transactionsEstimateFees.js

```diff
@@ -244,6 +246,30 @@ const calcAdditionalFees = async (transaction) => {
 	return additionalFees;
 };
 
+const validateTransactionParams = async transaction => {
+	// Mock optional values if not present before schema validation.
+	if (transaction.module === MODULE.TOKEN
+		&& transaction.command === COMMAND.TRANSFER_CROSS_CHAIN) {
+		if (!('messageFee' in transaction.params)) {
+			transaction.params.messageFee = '10000000';
+		}
+
+		if (!('messageFeeTokenID' in transaction.params)) {
+			transaction.params.messageFeeTokenID = '0000000000000000';
+		}
+	}
+
+	const allSchemas = await getSchemas();
+	const { schema: txParamsSchema } = allSchemas.data.commands.find(e => e.moduleCommand === `${transaction.module}:${transaction.command}`);
```

Comment:
will we get an error here if no schema value matches with the user supplied `transaction.module` and `transaction.command`?